sqlstore/src/main/resources/db/migration/V0010__sample_hierarchy.sql

```diff
@@ -334,3 +334,85 @@ CREATE TABLE `SampleTissue` (
+	(12,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(13,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(14,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(15,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v3 1×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v3 1×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v3 2×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v3 2×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v4 1×136', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'v4 2×126', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'Rapid Run 1×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'Rapid Run 1×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'Rapid Run 2×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(16,'Rapid Run 2×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(17,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(18,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(19,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(20,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(21,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(22,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(23,'Default', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(24,'1×300', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(24,'1×50', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(24,'1×500', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(24,'2×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(24,'2×151', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(24,'2×250', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(24,'2×26', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(24,'2×36', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(24,'300×200', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'v3 1×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'v3 1×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'v3 2×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'v3 2×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'v4 1×136', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'v4 2×126', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'Rapid Run 1×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'Rapid Run 1×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'Rapid Run 2×101', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP),
+	(25,'Rapid Run 2×51', 1, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP);
+
+CREATE TABLE `PoolOrder` (
+  `poolOrderId` bigint(20) PRIMARY KEY AUTO_INCREMENT,
+  `poolId` bigint(20) NOT NULL,
+  `partitions` int NOT NULL,
+  `parameterId` bigint(20) NOT NULL,
+  `createdBy` bigint(20) NOT NULL,
+  `creationDate` datetime NOT NULL,
+  `updatedBy` bigint(20) NOT NULL,
+  `lastUpdated` datetime NOT NULL,
+  -- Because Pool is MyISAM, we can't: CONSTRAINT `order_poolId_fkey` FOREIGN KEY (`poolId`) REFERENCES `Pool` (`poolId`),
```

Comment:
We've been converting to InnoDB as necessary to use FK references in Sample Hierarchy (see top of this file)